### DISCUSSION TITLE
Archive list has dropped some content and sizing now reflected in KB only since upgrading to 0.8.11

### DISCUSSION THREAD

**Discussion by rxgh99** [borgbase/vorta#1733] [FAQ] [answered]
Hi, 

Since upgrading to 0.8.11 I've noticed a change in my Archives listings:

-- The Size column now displays in KB only, making the sizing a little difficult to determine :) 

-- I've lost Size and Duration values for some archive records. This appears to be for older archives: after about 40-50 archives, the values are lost (values for the Date and Name columns remain). 

I've randomly selected some of these archives without Size and Duration present and it appears Extract works ok on them. 

I've attempted to refresh the repo; prune and reinstall Vorta, to no avail. 

Any ideas on what might have caused this and how to recover? 

Thanks!
![vorta-size-missing](https://github.com/borgbase/vorta/assets/81989963/65526e30-d4c9-4942-9761-6e3f17de3a72)

**Comment by real-yfprojects** [comment, marked as answer]
> I've attempted to refresh the repo; prune and reinstall Vorta, to no avail.
> 
> Any ideas on what might have caused this and how to recover?
> 
> 

I don't know what caused this but try running the archive specific refresh which you can find as a button on the right of the archive table.

**Reply by rxgh99** [reply]
Thanks @real-yfprojects 

On initial run, the archive-specific refreshing appears to be working - a couple of them have now finished and are showing data in the missing columns. However, the refresh takes a _long_ time to complete for these archives vs. those with no missing data in the columns e.g. multiple minutes vs. ~40 seconds.

**Reply by real-yfprojects** [reply]
> However, the refresh takes a _long_ time to complete for these archives vs. those with no missing data in the columns e.g. multiple minutes vs. ~40 seconds.

I guess that's a *borg* thing.

**Comment by real-yfprojects** [comment]
> The Size column now displays in KB only, making the sizing a little difficult to determine :)
> 
> 

This is already tracked by #1713.
